docs/celestial.rst

```diff
@@ -174,13 +187,31 @@ order of the interpolation. Supported strings include:
 * ``'nearest-neighbor'``: zeroth order interpolation
 * ``'bilinear'``: first order interpolation
 
+Additionally, one can control the calculation of the Jacobian used in this
+algorithm with the ``center_jacobian`` flag. The Jacobian matrix represents
+how the corresponding input-image coordinate varies as you move between output
+pixels (or d(input image coordinate) / d(output image coordinate)), and serves
+as a local linearization of the coordinate transformation. When this flag is
+True, the Jacobian is calculated at pixel grid points by calculating the
```

Comment:
Can you write True and False in this paragraph with backticks to get proper syntax highlighting/linking?